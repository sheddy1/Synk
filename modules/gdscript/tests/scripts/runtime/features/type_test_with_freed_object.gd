func test():
	var obj := Object.new()
	print(obj is Object)
	obj.free()
	print(obj is Object)
