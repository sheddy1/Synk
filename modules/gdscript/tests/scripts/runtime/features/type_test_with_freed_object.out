GDTEST_OK
true
false
